src/acl/Acl.h

```diff
@@ -73,7 +75,8 @@ class ACL
     virtual bool empty() const = 0;
     virtual bool valid() const;
 
-    int cacheMatchAcl(dlink_list * cache, ACLChecklist *);
+    using proxy_auth_match_cache = std::list<acl_proxy_auth_match_cache_entry>;
```

Comment:
Please use CamelCase for new names, including proxy_auth_match_cache and acl_proxy_auth_match_cache_entry. 

When converting to proper case, please place both names inside the `Acl` namespace (and outside the ACL class declaration). Besides being the Right Thing to do, it would fix your "document as code" [trick](https://github.com/squid-cache/squid/pull/590#discussion_r422553575) by giving both names the _same_ prefix.

Reply:
ok, update coming up